docs/resources/remote_site_id_mappings.md

```diff
@@ -0,0 +1,139 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "Multi-Site"
+layout: "aci"
+page_title: "ACI: aci_remote_site_id_mappings"
+sidebar_current: "docs-aci-resource-aci_remote_site_id_mappings"
+description: |-
+  Manages ACI Remote Site Id Mappings
+---
+
+# aci_remote_site_id_mappings #
+
+Manages ACI Remote Site Id Mappings
+
+
```

Comment:
should we add a note/warning with some additional information regarding the class since this is not a documented class and used only for specific use-case, which can be dangerous when mis-used?